playout/libretime_playout/liquidsoap/entrypoint.py

```diff
@@ -54,10 +53,8 @@ def generate_entrypoint(log_filepath: Optional[Path]):
             ss = legacy_client.get_stream_setting()
             generate_liquidsoap_config(ss, log_filepath)
             successful = True
-        except Exception as e:
-            print("Unable to connect to the Airtime server.")
-            logger.error(str(e))
-            logger.error("traceback: %s", traceback.format_exc())
+        except Exception:
+            logger.exception("Unable to connect to the Airtime server")
```

Comment:
Should we log the exception traceback?

Reply:
The traceback will always be present in the context of an exception handling. The string parameter will be title of the log entry with the traceback below.

```py
try:
    data["something"]
except KeyError:
    logger.exception(f"something key is missing from data")
```
Gives
```
$ python3 ./test.py   
2022-08-09 20:34:18.948 | ERROR    | __main__:<module>:12 - something key is missing from data
Traceback (most recent call last):

> File "/home/jo/git/github.com/libretime/libretime/shared/./test.py", line 10, in <module>
    data["something"]
    └ {}

KeyError: 'something'
```

```py
try:
    data["something"]
except KeyError as exception:
    logger.exception(f"something key is missing from data: {exception}")
```
Gives
```
$ python3 ./test.py
2022-08-09 20:34:41.934 | ERROR    | __main__:<module>:12 - something key is missing from data: 'something'
Traceback (most recent call last):

> File "/home/jo/git/github.com/libretime/libretime/shared/./test.py", line 10, in <module>
    data["something"]
    └ {}

KeyError: 'something'
```

I've been using the 2nd way in many other places, probably because it gives better context to the log message.

Reply:
Do you think we should standardise on a way to do this?

Reply:
I don't think we need a "standard" way of doing this, it should be handled case by case.

Once we removed all the `broad exception` ™  to specific exceptions, we will be able to know whether an exception message should be printing within the log entry or not, to print the traceback or not. But for now I have to admit it is difficult.